tests/queries/skip_list.json

```diff
@@ -106,5 +106,175 @@
     "polymorphic-parts": [
         "01508_partition_pruning", /// bug, shoud be fixed
         "01482_move_to_prewhere_and_cast" /// bug, shoud be fixed
+    ],
+    "parallel":
+    [
+        /// Pessimistic list of tests which work badly in parallel.
+        /// Probably they need better investigation.
+        "00062_replicated_merge_tree_alter_zookeeper",
```

Comment:
Do you mind sorting the list? (select + `:%sort` in vim). I go around sorting every list I see and often I find duplicate entries.

Didn't expect it to be so big btw, our tests are bad.... Hopefully having a parallel mode by default will make it harder to write bad tests. What about 'Flaky check', can it be made to run the same test concurrently, or is it a too hard condition?